src/udev/memory_id/dmidecode-config.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+/*
+ * Configuration
+ */
+
+#pragma once
+
+/* Use memory alignment workaround or not */
+#ifdef __ia64__
+#define ALIGNMENT_WORKAROUND
+#endif
```

Comment:
Is there any need to keep this in a header? It's only used in `dmidecode.c` afaict.

Reply:
There's really no need for rhetorical questions about the coding style. Nearly everything you see comes from dmidecode, which very much looks like 1999 code. Just ask me to make the changes, and I'll do that, although I'm still waiting for feedback on whether the approach is the correct one...